samples/nrf9160/nrf_cloud_rest_device_message/src/main.c

```diff
@@ -344,6 +346,49 @@ static int setup(void)
 	return 0;
 }
 
+int send_hello_world_msg(void)
+{
+	int err;
+	cJSON *msg_obj = NULL;
+
+	/* Send off a hello world message! */
+	/* This does not match a predefined schema. But that's not a problem! */
+	msg_obj = cJSON_CreateObject();
+	if (msg_obj == NULL) {
+		LOG_ERR("Failed to create cJSON object");
+		return -EINVAL;
+	}
+
+	if (!cJSON_AddStringToObject(msg_obj, "sample_message",
+					"Hello World, from the REST Device Message Sample!")) {
+		LOG_ERR("Failed to add sample message to cJSON object");
+		return -EINVAL;
+	}
+
+	if (IS_ENABLED(CONFIG_REST_DEVICE_MESSAGE_TEST_BUILD))	{
```

Comment:
I would think the timestamp would be nice whether in test mode or not.

The `CONFIG_REST_DEVICE_MESSAGE_TEST_BUILD` should still be used for the RTT console changes to simulate a button press, once you add that.